src/index.js

```diff
@@ -1,14 +1,15 @@
+import Router from 'next/router'
```

Comment:
We should receive Router as argument on line 5 and then pass it too withGa and withFBQ.

When trying to import next/router directly on the dependency caused Router to be undefined always.

Reply:
Completley agreed! I've updated the PR passing the router as a param instead of a dependency.